manifests/host.pp

```diff
@@ -9,6 +9,9 @@
   Boolean $ignored                      = false,
   Optional[Integer] $default_lease_time = undef,
   Optional[Integer] $max_lease_time     = undef,
+  Optional[String[1]] $ipxe_filename    = undef,
+  Optional[String[1]] $ipxe_bootstrap   = undef,
+  Optional[String[1]] $filename         = undef,
```

Comment:
Having parameter documentation for every newly added parameter would be nice.

Reply:
Where shall I put down the documentation?

Reply:
Ideally as puppet-strings inside this class. So
```puppet
# @summary Define a DHCP host
#
# @param ipxe_filename ...
# @param ipxe_bootstrap ...
define dhcp::host (
```

I won't make you document the existing parameters (though that would be awesome), but if we can at least document new parameters we can eventually get to a point where we can generate a useful reference documentation.

Reply:
I just pushed the parameter documentation.